packages/cli/src/commands/teams/command.ts

```diff
@@ -80,9 +68,15 @@ export const teamsCommand = {
       name: 'Invite multiple members simultaneously',
       value: `${packageName} teams invite abc@vercel.com xyz@vercel.com`,
     },
-    {
-      name: 'Paginate results, where `1584722256178` is the time in milliseconds since the UNIX epoch.',
-      value: `${packageName} teams ls --next 1584722256178`,
-    },
   ],
 } as const;
+
+export const teamsCommand = {
+  name: 'teams',
+  aliases: ['switch', 'team'],
+  description: 'Manage Teams under your Vercel account',
+  arguments: [],
+  subcommands: [addSubcommand, listSubcommand, switchSubcommand],
```

Comment:
The `inviteSubcommand` seems to be missing.

Reply:
Nice catch (878419bf17c6f7d521f852a5776d61060cf874fa)!